pytorch/inference/docker/2.2/py3/Dockerfile.graviton.cpu

```diff
@@ -0,0 +1,253 @@
+ARG PYTHON=python3
+ARG PYTHON_VERSION=3.10.9
+ARG MAMBA_VERSION=22.11.1-4
```

Comment:
I'd recommend upgrading mamba to at least `23.*.*` like we have in other PT DLCs [here](https://github.com/aws/deep-learning-containers/blob/f237519a5e6f1dcf9204610bab167f8417e235ca/pytorch/training/docker/2.2/py3/cu121/Dockerfile.gpu#L4). The latest version is now `24.*.*` so don't want to put ourselves in 2 major versions behind. You can try upgrading to v24 but at the very least we know v23 has no issues.

You can get the releases from [here](https://github.com/conda-forge/miniforge/releases)

Reply:
done